apps/emqx/src/emqx_misc.erl

```diff
@@ -546,3 +547,11 @@ ipv6_probe_test() ->
     end.
 
 -endif.
+
+pub_props_to_packet(Properties) ->
```

Comment:
I think it also makes sense to move https://github.com/kraftwerk28/emqx/blob/master/apps/emqx_connector/src/mqtt/emqx_connector_mqtt_mod.erl#L251-L272 and https://github.com/kraftwerk28/emqx/blob/master/apps/emqx_rule_engine/src/emqx_rule_events.erl#L1035-L1062 here, what do you think?

Reply:
maybe create a `emqx_message:props_to_map()`